subprojects/docs/src/docs/release/notes.md

```diff
@@ -50,6 +50,27 @@ Example:
 ADD RELEASE FEATURES BELOW
 vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv -->
 
+<a name="SSL"></a>
+### Faster Java compilation
+
+When compiling Java code, Gradle uses worker processes to run the Java compiler as a compiler daemon. 
+Compiler daemons are reused within a single build invocation. 
+This allows the Java compiler to "warm-up" and compile faster after a few uses. 
+
+In previous Gradle releases, the compiler daemons were always shut down at the end of the build.
+This caused every build to incur the cost of starting the compiler daemon and warming up the compiler.
+These start-up costs contributed to a large portion of overall build time when incrementally compiling a few source files.
+
+Starting in this release, Gradle attempts to keep Java compiler daemons around after the end of the build, so that subsequent builds are faster.
+Gradle will stop compiler daemons when too many workers are running. 
+Compiler daemons may also stop during a build if other worker processes are started and Gradle needs to free up resources.
+
+No configuration changes are required to enable this feature. 
+
+<!-- TODO: Link to blog post that details the measurements -->
+
+Persistent compiler daemons for Groovy, Scala or Kotlin will be evaluated in the future.
```

Comment:
Does it make sense to mention Kotlin here? Wouldn't that more be up to us exposing a flag to control this in public worker API?

Reply:
I was thinking from a casual user's POV, all of the languages are the same, even if they're implemented by different groups or APIs.

This might be misleading for another reason... I think Kotlin already has a persistent compiler daemon. I'm not sure if it's on by default.